wof/WaterML_1_1.py

```diff
@@ -4605,9 +4605,9 @@ def validate_definitionType(self, value):
         # Validate type definitionType, a restriction on xsi:string.
         if value is not None and Validate_simpletypes_:
             if len(value) > 255:
-                warnings.warn(u'Value "%(value)s" does not match xsd maxLength restriction on definitionType' % {"value" : value.encode("utf-8")} )
+                warnings.warn(u'Value "%(value)s" does not match xsd maxLength restriction on definitionType' % {"value" : value} )
             if len(value) < 0:
-                warnings.warn(u'Value "%(value)s" does not match xsd minLength restriction on definitionType' % {"value" : value.encode("utf-8")} )
+                warnings.warn(u'Value "%(value)s" does not match xsd minLength restriction on definitionType' % {"value" : value} )
```

Comment:
If this works we have a single `.decode` point at

```
def quote_xml_aux(inStr):
    inStr = inStr.decode('utf-8')
    s1 = inStr.replace('&', '&amp;')
    s1 = s1.replace('<', '&lt;')
    s1 = s1.replace('>', '&gt;')
    return s1
```

and no re-encoding. It should be all unicode/str (py2/py3).

We may still write it as bytes in the future.